cfgmgr/scheduledconfigmgrd.cpp

```diff
@@ -0,0 +1,76 @@
+#include <unistd.h>
+#include <vector>
+#include <mutex>
+#include "dbconnector.h"
+#include "select.h"
+#include "exec.h"
+#include "schema.h"
+#include "scheduledconfigmgr.h"
+#include <fstream>
+#include <iostream>
+
+using namespace std;
+using namespace swss;
+
+/* SELECT() function timeout retry time, in millisecond */
+#define SELECT_TIMEOUT 1000
```

Comment:
why do we need to "wake up" every second?

Reply:
The timeout will end up calling the doTask() function again with all the consumables still in the queue. This is usually used for "retrying" tasks that did not succeed the first time, but select will not return because no changes have been made to the DB to trigger it again.